src/components/DocumentationTopic.vue

```diff
@@ -32,8 +32,7 @@
           :objcPath="objcPath"
           :swiftPath="swiftPath"
         />
-        <LinkableHeading v-if="enableMinimized" class="minimized-summary">Summary</LinkableHeading>
-        <Title v-else :eyebrow="roleHeading">
+        <Title :eyebrow="roleHeading" :enableMinimized="enableMinimized">
```

Comment:
I think the eyebrow is already an optional prop to the title component so you can just omit that in the minimized case and apply a special class here for the minimized title similar to how you're already doing it for the description/abstract in other places within this file—that should avoid the need to pass this minimized flag to deeper components and have it simply be a feature of this one.